policyengine_us/parameters/gov/states/az/hhs/tanf/eligibility/payment_standard/low.yaml

```diff
@@ -0,0 +1,13 @@
+description: Arizona provides the following low percentage of FPG as cash assistance payment standard if the filer does not pay shelter costs, based on the family size.(might have rounding error below 3)
+
+metadata:
+  unit: /1
+  period: month
+  label: Arizona cash assistance lower payment standard
+  reference:
+    - title: Arizona Department of Economic Security - Cash Assistance (CA) Income Eligibility Guidelines
+      href: https://des.az.gov/services/child-and-family/cash-assistance/cash-assistance-ca-income-eligibility-guidelines
```

Comment:
add a reference that shows the calculated values and add a comment:
"# Source {xyz} rounds the results to a percentage of 22.65 instead of 23%, which is defined in the manual"